src/bin/server.rs

```diff
@@ -33,12 +60,43 @@ async fn main() -> anyhow::Result<(), Box<dyn std::error::Error>> {
         .filter(None, log::LevelFilter::Debug);
     builder.init();
 
-    //todo
-    //read from command line.
+    // read from default configuration.
+    let default_yaml_str = include_str!("../../examples/server.yaml");
```

Comment:
I think "/etc/sealfs/server.yaml" might be better

Reply:
Definitely. The reason why it's not modified is that testing will be more convenient. This will be fixed in the next commit.

Reply:
I think it's alright.